test/src/test/java/test/compatibility/BridgeMethodsTest.java

```diff
@@ -0,0 +1,36 @@
+package test.compatibility;
```

Comment:
Does not match https://github.com/jenkinsci/jenkins/blame/b36eec37598edfd20dc063e5380421befe1c03f0/test/src/test/java/test/README.md#L1 BTW.

Reply:
where would you suggest that tests go that do not check specific functionality?  
I expliclty did not want to put this in `hudson.model.QueueTest` or indeed in any `hudson.` `jenkins.` as it is not scoped to that level (even if it is using the Queue for testing purposes)

Reply:
Right…`jenkins` package? Or `jenkins.bugs`? No strong opinion.

Reply:
addressed in fc967a6ba40ebf23e134f27b4c651facf3fe6e9f